debezium-connector-db2/src/main/java/io/debezium/connector/db2/Db2SourceInfoStructMaker.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.db2;
+
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.Struct;
+
+import io.debezium.config.CommonConnectorConfig;
+import io.debezium.connector.AbstractSourceInfoStructMaker;
+
+public class Db2SourceInfoStructMaker extends AbstractSourceInfoStructMaker<SourceInfo> {
+
+    private final Schema schema;
+
+    public Db2SourceInfoStructMaker(String connector, String version, CommonConnectorConfig connectorConfig) {
+        super(connector, version, connectorConfig);
+        schema = commonSchemaBuilder()
+                .name("io.debezium.connector.db2.Source")
+                .field(SourceInfo.SCHEMA_NAME_KEY, Schema.STRING_SCHEMA)
+                .field(SourceInfo.TABLE_NAME_KEY, Schema.STRING_SCHEMA)
+                .field(SourceInfo.CHANGE_LSN_KEY, Schema.OPTIONAL_STRING_SCHEMA)
```

Comment:
Change and commit LSN, do they actually exist for DB2? The terms are so SQL Server specifcic?

Reply:
In DB2 are named commit LSN and sequences LSN